From 0f9ae3282a20e5877c5e8884d749559dc9d204f2 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Thu, 8 Dec 2016 21:09:03 +0100
Subject: [PATCH 2/2] include <stdlib.h> for putenv()

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
---
 t/05read.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/t/05read.c b/t/05read.c
index 073c43d..c9bbe93 100644
--- a/t/05read.c
+++ b/t/05read.c
@@ -1,5 +1,6 @@
 #include <stdio.h>
 #include <errno.h>
+#include <stdlib.h>
 #include <unistd.h>
 #include "../termkey.h"
 #include "taplib.h"
-- 
2.11.0

